kie-drools-wb-parent/kie-drools-wb-webapp/pom.xml

```diff
@@ -2053,6 +2053,31 @@
                 <classifier>redhat</classifier>
               </configuration>
             </plugin>
+            <plugin>
+            <groupId>org.wildfly.swarm</groupId>
+            <artifactId>wildfly-swarm-plugin</artifactId>
+            <configuration>
+              <fractionDetectMode>never</fractionDetectMode>
+              <fractions>
+                <fraction>logging</fraction>
+                <fraction>undertow</fraction>
+                <fraction>cdi</fraction>
+                <fraction>ejb</fraction>
+                <fraction>jaxrs</fraction>
+                <fraction>ee</fraction>
+                <fraction>security</fraction>
+                <fraction>management</fraction>
+                <fraction>datasources</fraction>
+              </fractions>
+            </configuration>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>package</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
```

Comment:
Could we move the config outside of the profiles? That would make sure it would be build even for -Dproductized. And it would save us some duplication.